atlasdb-commons/src/main/java/com/palantir/async/initializer/AsyncInitializer.java

```diff
@@ -34,21 +34,15 @@
  * In order to be ThreadSafe, the abstract methods of the inheriting class need to be synchronized.
  */
 @ThreadSafe
-public interface AsyncInitializer {
-    int millisUntilNextAttempt = 10_000;
-    Logger log = LoggerFactory.getLogger(AsyncInitializer.class);
-    int nThreads = 20;
-    ScheduledExecutorService executorService = Executors.newScheduledThreadPool(
-            nThreads, new NamedThreadFactory("AsyncInitializer", true));
+public abstract class AsyncInitializer {
+    private static final Logger log = LoggerFactory.getLogger(AsyncInitializer.class);
+    private static final int NUM_THREADS = 100;
```

Comment:
we should eventually make this not actually create 100 threads. ideally the executor service can be configured to not have 100 threads sitting around, and only create them if needed